packages/example-getting-started/docs/8-putting-it-together.md

```diff
@@ -1,61 +1,84 @@
+will automatically discover our controllers, repositories, datasources and
+other artifacts and inject them into our application for use.
+In the current example, we've manually loaded our repository so that it can be
+mocked/stubbed for unit testing purposes.
+
+>**NOTE**: The boot module will only discover and inject artifacts that
+follow our established conventions for artifact directories. Here are some
+examples:
+>* Controllers: `./src/controllers`
+>* Datasources: `./src/datasources`
+>* Models: `./src/models`
+>* Repositories: `./src/repositories`
 
 #### src/application.ts
 ```ts
 import {ApplicationConfig} from '@loopback/core';
 import {RestApplication} from '@loopback/rest';
-import {TodoController, PingController} from './controllers';
+import {TodoRepository} from './repositories';
+import {db} from './datasources/db.datasource';
+
+/* tslint:disable:no-unused-variable */
+// Do not remove!
+// Class and Repository imports required to infer types in consuming code!
+// Binding and Booter imports are required to infer types for BootMixin!
+import {BootMixin, Booter, Binding} from '@loopback/boot';
 import {
   Class,
   Repository,
-  RepositoryMixin,
   DataSourceConstructor,
+  RepositoryMixin,
 } from '@loopback/repository';
-import {db} from './datasources/db.datasource';
-import {TodoRepository} from './repositories';
+/* tslint:enable:no-unused-variable */
 
-export class TodoApplication extends RepositoryMixin(RestApplication) {
+export class TodoApplication extends BootMixin(
+  RepositoryMixin(RestApplication),
+) {
   constructor(options?: ApplicationConfig) {
     super(options);
-    this.setupControllers();
+    this.projectRoot = __dirname;
     this.setupRepositories();
   }
 
-  setupControllers() {
-    this.controller(TodoController);
-    this.controller(PingController);
-  }
-
+  // Helper functions (just to keep things organized)
   setupRepositories() {
-    // This will allow you to test your application without needing to
-    // use the "real" datasource!
     const datasource =
       this.options && this.options.datasource
         ? new DataSourceConstructor(this.options.datasource)
         : db;
     this.bind('datasource').to(datasource);
     this.repository(TodoRepository);
   }
-}
+}}
 ```
 
 ### Try it out
 
-Now that your app is ready to go, try it out with your favourite REST client!
-Start the app (`npm start`) and then make some REST requests:
-- `POST /todo` with a body of `{ "title": "get the milk" }`
-- `GET /todo/1` and see if you get your Todo object back.
-- `PATCH /todo/1` with a body of `{ "desc": "need milk for cereal" }`
+Let's try out our application!
+First, you'll want to start the app.
+```
+$ npm start
+Server is running on port 3000
+```
 
-### Navigation
+Next, you can use the [API Explorer](http://loopback.io/api-explorer/?url=http://localhost:3000/swagger.json) to browse your API and make requests!
```

Comment:
Change this link to `http://localhost:300/swagger-ui` so it's easier for users to understand and follow along.